### ISSUE TITLE
Missing file in VFS documentation

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4240]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4240|
|Type|defect|
|Created|2010-01-28 20:29:36Z|

`/vfs/index.xhtml` references  (links to) `/vfs/examples/index.html`, but the file does not exist.

Attachments:

* [4240_add_index_file.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/403/403667553498df159bdd52af68ccf072f564fcc0/026e9cc9be1b6deadc01d25a4051747102c98511.patch) (488 bytes) - added by khorn on 2010-01-28 20:39:44Z - Add missing index file

<details><summary>Searchable metadata</summary>

```
trac-id__4240 4240
type__defect defect
reporter__khorn khorn
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__vfs vfs
keywords__documentation documentation
time__1264710576000000 1264710576000000
changetime__1264716764000000 1264716764000000
version__None None
owner__khorn khorn
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_1"></a>|@khorn removed owner|
|-|-|

Attached patch adds the missing file.

It would be nice if someone who knows anything about VFS would:

 * Write some docs explaining these examples, or failing that...
 * Provide a blurb telling us what these examples are intended to do, or at least...
 * Comment the example files a bit

But I don't know that that would need to be on this ticket.

Anyway, up for review.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set owner to @khorn|
|-|-|

Please merge, thanks.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_3"></a>|@khorn set status to `closed`|
|-|-|

(In [28087]) Merge: Add /vfs/examples/index.xhtml

Author: khorn
Reviewers: thijs
Fixes: [#4240](https://github.com/twisted/twisted/issues/4240)

Add missing documentation file: /doc/vfs/examples/index.html
